wolfcrypt/src/port/Espressif/esp32_mp.c

```diff
@@ -146,18 +149,15 @@ static void process_start(word32 reg)
 /* wait until done */
 static int wait_uitil_done(word32 reg)
 {
-    int timeout = 0;
+    word32 timeout = 0;
     /* wait until done && not timeout */
-    while(1) {
-        if(++timeout < ESP_RSA_TIMEOUT && DPORT_REG_READ(reg) == 1){
-            break;
-        }
-    }
+    while(!ESP_TIMEOUT(++timeout) && 
+                DPORT_REG_READ(reg) != 1) { }
 
     /* clear interrupt */
     DPORT_REG_WRITE(RSA_INTERRUPT_REG, 1);
 
-    if(timeout >= ESP_RSA_TIMEOUT) {
+    if(ESP_TIMEOUT(timeout)) {
         ESP_LOGE(TAG, "rsa operation is time-outed.");
```

Comment:
As above, "time-outed." should change to "timed out."

Reply:
Thanks, fixed.